### DISCUSSION TITLE
Service_Code and Billing_code_modifier arrays

### DISCUSSION THREAD

**Discussion by zmalone91** [CMSgov/price-transparency-guide#587] [Q&A]
How should I interpret the arrays for **service_code** and **billing_code_modifier** in the negotiated_prices object example below? How is the negotiated_rate of 64.52 applied to the different combinations of service/modifier codes? 

                    "negotiated_prices": [
                        {
                            "negotiated_type": "derived",
                            "negotiated_rate": 64.52,
                            "expiration_date": "2023-01-01",
                            "service_code": [
                                "11",
                                "22"
                            ],
                            "billing_class": "professional",
                            "billing_code_modifier": [
                                "25",
                                "GC"
                            ],
                            "additional_information": "null"
                        }

Is the same rate given to each combination like so?:
![image](https://user-images.githubusercontent.com/34782953/185230405-034c1ef4-e19c-4057-b543-9e295a1b9f68.png)

**Comment by dgolden** [comment]
Your interpretation is close to mine. However, the rows with "null" should not be there. Your nulls presumably come from the "additional information" field, but that field is at the same hierarchy level as negotiated rate, billing code modifier, so "additional information" should appear as an additional column.

However to get your arms around this data, for your initial investigation, I would ignore "additional information", especially as it is not a required field per the [CMS schema specification](https://github.com/CMSgov/price-transparency-guide/tree/master/schemas/in-network-rates#provider-reference-object).

As you can see, normalizing the data to fit within the familiar confines of Excel/SQL/R, as you did here, leads to a tsunami of records.
